<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelDialog</class>
 <widget class="QDialog" name="modelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose table</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="modelListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
